test/unit/view.child-views.spec.js

```
@@ -33,7 +33,7 @@ describe('layoutView', function() {
     this.ViewNoDefaultRegion = this.View.extend({
       regions: {
         regionOne: {
-          selector: '#regionOne',
+          el: '#regionOne',
```

these tests failed silently before 😿